### ISSUE TITLE
"Cannot find module" on cdktf synth in 0.14.0

### ISSUE THREAD

**Issue by nbaju1** [hashicorp/terraform-cdk#2301]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### cdktf & Language Versions

- Python 3.9.0
- Node 16.17.0
- CDKTF 0.14.0

### Affected Resource(s)

Generating .json file from `cdktf synth` in 0.14.0. Works without issue in ~0.12. Python script has been updated with the provider specific imports required in 0.14.0. 

### Debug Output
https://gist.github.com/nbaju1/9bae75d35e0ea402b1bd469012e07549

### Expected Behavior

.json file being created.

### Actual Behavior

Fails on`cdktf synth`. See gist.

### Steps to Reproduce

```
npm install --global cdktf-cli@~0.14.0
pip install pipenv
pipenv install
cdktf synth
```

### References

- #2300

**Comment by ansgarm** [comment]
Hi @nbaju1 👋
See my comment over in a similar issue (with the same underlying reason): https://github.com/hashicorp/terraform-cdk/issues/2300#issuecomment-1316955350

I'll close this as a duplicate.

tl;dr: We're working on a fix!
